assets/src/components/minimalLadderPage.tsx

```diff
@@ -1,36 +1,26 @@
 import React, { useContext } from "react"
-import RouteLadders from "./routeLadders"
-import { currentRouteTab } from "../models/routeTab"
+import { isPreset } from "../models/routeTab"
 import { StateDispatchContext } from "../contexts/stateDispatchContext"
-import {
-  deselectRouteInTab,
-  flipLadderInTab,
-  toggleLadderCrowdingInTab,
-} from "../state"
+import { Button } from "react-bootstrap"
 
 export const MinimalLadderPage = () => {
-  const [{ routeTabs }, dispatch] = useContext(StateDispatchContext)
-
-  const { selectedRouteIds, ladderDirections, ladderCrowdingToggles } =
-    currentRouteTab(routeTabs) || {
-      selectedRouteIds: [] as string[],
-      ladderDirections: {},
-      ladderCrowdingToggles: {},
-    }
+  const [{ routeTabs }] = useContext(StateDispatchContext)
+  const presets = routeTabs
+    .filter(isPreset)
+    .sort((a, b) => (a.presetName || "").localeCompare(b.presetName || ""))
 
   return (
-    <div className="c-ladder-page__tab-bar-and-ladders">
-      <RouteLadders
-        selectedRouteIds={selectedRouteIds}
-        selectedVehicleId={undefined}
-        deselectRoute={(routeId) => dispatch(deselectRouteInTab(routeId))}
-        reverseLadder={(routeId) => dispatch(flipLadderInTab(routeId))}
-        toggleCrowding={(routeId) =>
-          dispatch(toggleLadderCrowdingInTab(routeId))
-        }
-        ladderDirections={ladderDirections}
-        ladderCrowdingToggles={ladderCrowdingToggles}
-      />
+    <div className="c-minimal-ladder-page">
+      <h3>Select a preset to display</h3>
+      {presets.map((preset) => (
+        <Button
+          className="c-minimal-ladder-page__button"
+          key={preset.uuid}
+          href={"/minimal/" + preset.uuid}
+        >
+          {preset.presetName}
+        </Button>
```

Comment:
Yes, I can do this with some variation: the reactstrap Button doesn't like the prop `as`, so instead I can wrap the Button in a NavLink.

If I use the navless Button, the UX is that the page is blank for a moment before the route ladders render fully loaded. If I use the NavLink approach, the page renders kinda wonky before filling out. What is the benefit to going with NavLink here?

![May-13-2024 17-21-44](https://github.com/mbta/skate/assets/69368883/4163225d-b8cf-4920-9dec-d41556abaca5)

Reply:
1. I realize we don't _need_ `NavLink` specifically, but I'd like to use the [`Link` from `reactrouter`](https://reactrouter.com/en/main/components/link) since this is a url that is managed by reactrouter.
2. Is the difference in UX shown in the video posted above? I'm having a hard time seeing the difference

Reply:
Ended up sketching out an idea https://github.com/mbta/skate/pull/2588#discussion_r1600363159

Reply:
Also I tried the UX of a regular link vs a react router link, and I feel that the loading is smoother because it's a SPA transition when using `<Link>`, but it's a regular HTTP request when using `<Button href={} />` which ends up doing full page layout and first time rendering again.

Reply:
Ahhhh ok. I see why the react router link choppiness is still a positive tradeoff, then. Thanks! 👍